### DISCUSSION TITLE
Welcome to Ark Ascended Tools Discussions!

### DISCUSSION THREAD

**Discussion by Kexodus1** [Kexodus1/Ark-Ascended-Tools#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sudo-q00us** [comment]
Club ark thing didnt work or need to put your keyboard on azerty or querty ?

**Reply by sudo-q00us** [reply]
also nothing work on club ark script
